sched/sagaLog.thrift

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Suggestion: Rename JobDef.thrift or Definitions.thrift.  This isn't necessarily specific to the SagaLog.  

The SagaLog just stores opaque bytes doesn't care about the format.  Also this serializer/deserializer will be used to place jobs on the Work Queue too.